.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,31 @@
+name: gh-pages
+on:
```

Comment:
The `on` section allows us to trigger workflow depending on developer action and on specific branches. Currently, our setup will trigger the workflow when someone pushes (merges) changes to `master` or makes a PR to `master`.